packages/react-server/src/ReactFlightServer.js

```diff
@@ -1836,6 +1835,15 @@ function serializeLimitedObject(): string {
   return '$Y';
 }
 
+function serializeHaltedReference(): string {
+  return '$&';
+}
+const reusableHaltedReferenceModel = '"$&"';
+
+function serializeLazyHaltedReference(): string {
+  return '$&L';
```

Comment:
Shouldn't it be:
```suggestion
  return '$L&';
```
Assuming `&` is for halted references what otherwise would be the ID.

Reply:
Lazy will be super common so I don’t want a special I’d check in that pathway. Halts will be relatively rare so forking the impl in the client there is less of a concern. Either way these need their own special handling, in fact I could do something like `”$?”` for this type and just avoid a second condition check but I don’t think it is super important and the L at least provides some cognitive association with lazy wrapped that I thought I’d go with this

Reply:
Thought about it more and when prerendering there will be lots of lazies and even if the eager type is more common it will need to do this conditional check here so I changed the lazy one to `"&*"`